starter-code/lodash.js

```diff
@@ -0,0 +1,203 @@
+    }
+
+console.log("Test - Different Characters(goodPsswd1): ", noRepeatChar(goodPsswd1));
+console.log("Test - Different Characters(badPsswd1): ", noRepeatChar(badPsswd1));
+
+
+// Only numbers
+
+// JS Vanilla & ES6
+
+const goodPsswd2 = "1234567890";
+const badPsswd2 = "1a234567890";
+const onlyNumbers2 = (password) => {
+    const reg = /^\d+$/;
+    return reg.test(password);
+}
+// console.log(onlyNumbers(goodPsswd4));
+//console.log(onlyNumbers(badPsswd4));
+
+const onlyNumbers3 = (password) => {
+    return _.toNumber(password) ? "The password is ok" : "Your password is not OK";
+}
+console.log("Test - Check a OK Password (goodPsswd2): ", onlyNumbers3(goodPsswd2));
+console.log("Test - Check a NOK Password (badPsswd2): ", onlyNumbers3(badPsswd2));
+
+// Ten digits only
+const goodPsswd = "1234567890";
+const badPsswd = "12345678901234567890";
+const trimPassword = (password) => _.truncate(password, { 'length': 10, 'omission': '' });
+console.log("Trim Password to 10 digits (badPsswd): ", trimPassword(badPsswd));
+
+
+//////////////////////////////////////////
+/////////////////////////////////////////
+// Iteration 4 - Abbey Road Studios /////
+////////////////////////////////////////
+////////////////////////////////////////
+
+// November looks like a good month:
+const novemberArtists = () => {
+    monthNovember = _.filter(AbbeyRoad, { 'month': 11 });
+    const artists = _.mapValues(monthNovember, 'artist');
+    const array = _.map(artists, (value, index) => [value]);
+    const counter = _.countBy(array);
+    const result  = _(array).uniq().sortBy(item => counter[item]).reverse().value();
+    return result[0][0];
+};
+
+console.log("Artist who recorded the most on November in Abbey Road: ", novemberArtists());
+
+
+// Artists like to repeat:
+const bestArtist = () => {
+    const artistCount = _.countBy(AbbeyRoad, "artist");
+    let test = 0;
+
+    for (const key in artistCount) {
+        if(test < artistCount[key]) {
+            test = artistCount[key];
+        }
+    }
+    return _.findKey(artistCount, _.partial(_.isEqual, test));
+};
+
+console.log("Artist who recorded the most times in Abbey Road: ", bestArtist());
+
+
+// The Beatles and Abbey Road:
+const lastBeatlesSong = () => {
+    const songs = _.filter(AbbeyRoad, (el) => _.has(el, 'artist') && el.artist === 'The Beatles');
+	return _.maxBy(songs, 'year').year;
+};
+
+console.log("Last song in Abbey Road Studios: ", lastBeatlesSong());
+
+
+// Sixties crazyness:
+const sixtiesSong = () => {
+    const songsDecade1960 = _.filter(AbbeyRoad, (el) => _.has(el, 'year') && el.year < 1970 && el.year > 1959);
+	//console.log(songsDecade1960);
+    const songsDecade1960Sorted = _.sortBy(songsDecade1960, ['year', 'month']);
+    //console.log(songsDecade1960Sorted);
+    return _.last(songsDecade1960Sorted).song;
+};
+
+console.log("Last song in the 1960's: ", sixtiesSong());
```

Comment:
Very well done, you implemented the lodash methods in the proper way and I like you are getting used to reg expressions